TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/TrsLinkGenerator.cs

```diff
@@ -27,7 +29,9 @@ public TrsLinkGenerator(LinkGenerator linkGenerator)
 
     public string Persons() => GetRequiredPathByPage("/Persons/Index");
 
-    public string PersonDetail() => GetRequiredPathByPage("/Persons/PersonDetail/Index");
+    public string PersonDetail(Guid personId, string? selectedTab = null) =>
```

Comment:
Could we use the `PersonSubNavigationTab` enum instead of `string`?

Reply:
Doh .. yeah of course!

Reply:
Done